components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -140,7 +140,7 @@ const ContributorProfilePage = ({
                   </div>
                 </div>
                 <div className="mt-10 h-32">
-                  <CardLineChart lineChartOption={chart} />
+                  <CardLineChart lineChartOption={chart} className="!h-32" />
```

Comment:
The important hack here needs described in the PR description, otherwise it can be missed 🍕

Reply:
Sure